dsa-solutions/lc-solutions/2200-2299/2241-Design-an-ATM-Machine.md

```diff
@@ -0,0 +1,268 @@
+---
+id:  Design an ATM Machine
```

Comment:
Please remember to change `id: Design an ATM Machine` to `id: design-an-atm-machine`